magenta/protobuf/music_pb2.py

```diff
@@ -3,11 +3,13 @@
 # source: music.proto
 
 import sys
-_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
+
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import message as _message
 from google.protobuf import reflection as _reflection
 from google.protobuf import symbol_database as _symbol_database
+
+_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
```

Comment:
How sure are you that this doesn't break anything?  Is there any chance the above imports were somehow reading this variable?

Reply:
reverted